src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/ChestSlimefunGuide.java

```diff
@@ -96,7 +96,7 @@ public boolean isSurvivalMode() {
         return true;
     }
 
-    private List<Category> getVisibleCategories(Player p, PlayerProfile profile) {
+    protected List<Category> getVisibleCategories(Player p, PlayerProfile profile) {
```

Comment:
A javadocs tag and some nullability annotations would be nice.
Other than that it's looking good as it is.